package.json

```
@@ -21,6 +21,7 @@
     "generate": "npm-run-all generate:*",
     "generate:definitions-internal": "rimraf ./generated/definitions/internal && shx mkdir -p ./generated/definitions/internal && gen-api-models --api-spec ./openapi/internal.yaml --no-strict --out-dir ./generated/definitions/internal",
     "generate:definitions-external": "rimraf ./generated/definitions/external && shx mkdir -p ./generated/definitions/external && gen-api-models --api-spec ./openapi/external.yaml --no-strict --out-dir ./generated/definitions/external",
+    "generate:definitions-hub-spid-login": "rimraf ./generated/definitions/hub-spid-login && shx mkdir -p ./generated/definitions/hub-spid-login && gen-api-models --api-spec ./openapi/hub-spid-login.yaml --no-strict --out-dir ./generated/definitions/hub-spid-login --request-types --response-decoders --client",
```

This command generate even the client not only the definitions, we should rename the command accordingly.
```suggestion
    "generate:hub-spid-login": "rimraf ./generated/definitions/hub-spid-login && shx mkdir -p ./generated/definitions/hub-spid-login && gen-api-models --api-spec ./openapi/hub-spid-login.yaml --no-strict --out-dir ./generated/definitions/hub-spid-login --request-types --response-decoders --client",
```